osp-backend-docusaurus/docs/Code- Organization.md

```diff
@@ -0,0 +1,15 @@
+---
+id: Code-Organization
```

Comment:
There are two files present with name `Code- Organization.md` and `Code-Organization.md`. Please remove the first one as that is not needed.

Reply:
Yes I just noticed that I was just renaming the file & I guess I added it twice I will remove it